hw4/02-sessions.js

```diff
@@ -1,16 +1,47 @@
 const express = require('express');
 const session = require('express-session');
 const app = express();
-const port = process.env.PORT || 5000;
+const port = process.env.PORT || 5001;
 
 // Add your code here
 
 // Use the express-session module
-app.use(/** ... */);
+app.use(
+  session({
+  store: new session.MemoryStore(),
+  secret: 'a secret to sign the cookie',
+  resave: false,
+  saveUninitialized: false,
+  cookie: {
+    maxAge: 86400000,
+  },
+})
+);
 
-app.get('/', (req, res) => {
+app.get('*', (req, res) => {
   // Add your code here
-});
+  res.status(200);
+  res.set({'Content-Type': 'text/plain'});
+  if (req.session.visit === undefined){
+    req.session.visit = [];
+    req.session.visit.push('/');
```

Comment:
This line assumes that the user launches the home page directly. This ignores the case when user visits the website for the first time through a different URL. 
For eg. If user enters `localhost:5001/firstVisit ` it would still save the history as `/` instead of `/firstvisit`